src/SLCore.Listeners/Implementation/Analysis/RaisedFindingProcessor.cs

```diff
@@ -78,12 +78,6 @@ private bool IsValid<T>(RaiseFindingParams<T> parameters) where T : RaisedFindin
             return false;
         }
 
-        if (parameters.isIntermediatePublication)
```

Comment:
With the change to IssueConsumer, we can stop ignoring the intermediate events. This allows us to have issue streaming from SLCore